content/post/practical-outbreakend.Rmd

```diff
@@ -0,0 +1,408 @@
+
+The important result of @nishiura_etal2016 is that the probability $\pi_t = P(Y_t > 0\>|\>D)$ for $t=n+1,n+2,\ldots$ can be computed as follows:
+$$
+\begin{align*}
+\pi_t = 1 - \prod_{i=1}^n \sum_{o=0}^{\infty} f_{\text{offspring}}(o; R_0, k) \cdot \left[ F_{\text{serial}}(t-t_i) \right]^{o},
+\end{align*}
+$$
+where $f_{\text{offspring}}$ denotes the probability mass function (PMF) for the number of secondary cases that one primary case induces. It is assumed that this distribution is negative binomial with expectation $R_0>0$ and clumping parameter $k>0$. In other words, $\operatorname{E}(O)=R_0$ and $\operatorname{Var}(O)=R_0 + R_0^2/k$. 
+Furthermore, $F_{\text{serial}}$ denotes the CDF of the serial interval distribution of the disease of interest. The serial interval is the time period between the onset of symptoms in the primary and onset of symptoms in the secondary case, see @svensson2007 for details and definitions. 
+
+Once $\pi_t$ is below some pre-defined threshold $c$, say $c=0.05$, one would declare the outbreak to be over, if no new cases have been observed by time $t$. In other words:
+$$
+T_{\text{end}} = \min_{t>n} \{ \pi_t < c \}.
+$$
+
+Note that the formulated approach is conservative, because every available case is treated as having the potential to generate new secondary cases according to the entire offspring distribution. In practice, however, observed cases towards the end will be secondary cases of some of the earlier cases. Hence, these primary cases will be attributed as having the ability to generate more secondary cases than they actually have in practice. Another important assumption of the method is that all cases are observed: no asymptomatic cases nor under-reporting is taken into account.
+   
+## Required packages
+
+The following packages, available on CRAN, are needed for this practical:
+
+- [`tidyverse`](http://tidyverse.tidyverse.org) A set of packages (aka. the *hadleyverse*) to enhance the necessary data munging
+- [`openxlsx`](https://cran.r-project.org/web/packages/openxlsx/index.html) to read `.xlsx` files without `rJava` dependence
```

Comment:
`readxl` is also a very good package that I always use.

Reply:
I agree, however, it leads to all column names with spaces being converted into columns names like ``Date of notification`and the type guessing of the integer cells doesn't appear to work when using:

linelist <- readxl::read_xlsx("../../static/data/MERS-CoV-cases-rok-21Jul15.xlsx",skip=3)

This works better with xlsx/openxlsx. Since the other practicals use the xlsx package, I found it ok to use this package. Therefore: No changes made at this point.